src/runtime_src/core/tools/common/SubCmdExamineInternal.cpp

```diff
@@ -103,8 +103,25 @@ SubCmdExamineInternal::execute(const SubCmdOptions& _options) const
   XBU::verbose("SubCommand: examine");
 
   // Parse sub-command ...
-  po::variables_map vm;
-  process_arguments(vm, _options);
+  try{
+    po::variables_map vm;
+    const auto unrecognized_options = process_arguments(vm, _options, false);
+
+    if (!unrecognized_options.empty())
+    {
+      std::string error_str;
+      error_str.append("Unrecognized arguments:\n");
+      for (const auto& option : unrecognized_options)
+        error_str.append(boost::str(boost::format("  %s\n") % option));
+      throw boost::program_options::error(error_str);
+    }
+  }
+  catch (boost::program_options::error& e)
```

Comment:
const?

Reply:
There is a top level catch in **SubCmd::process_arguments** that used to catch this earlier. But that catch printed a generic help (ryzenAI + Alveo) which was incorrect. Now this dedicated catch will catch this and print specific help based on the device. 
The const part is fixed now.